hal/transport/RFM95/MyTransportRFM95.cpp

```diff
@@ -48,11 +70,26 @@ uint8_t transportGetAddress(void)
 
 bool transportSend(const uint8_t to, const void* data, const uint8_t len, const bool noACK)
 {
+#if defined(MY_RFM95_ENABLE_ENCRYPTION)
+	// copy input data because it is read-only
+	(void)memcpy(RFM95_dataenc,data,len);
+	// has to be adjusted, WIP!
+	RFM95_aes.set_IV(0);
+	const uint8_t finalLength = len > 16 ? 32 : 16;
+	//encrypt data
+	RFM95_aes.cbc_encrypt(RFM95_dataenc, RFM95_dataenc, finalLength /16);
+	if (noACK) {
+		(void)RFM95_sendWithRetry(to, RFM95_dataenc, len, 0, 0);
+		return true;
+	}
+	return RFM95_sendWithRetry(to, RFM95_dataenc, len);
```

Comment:
same here, finalLength

Reply:
Pretty sure I had this at some point, but a rebase might have nuked it. Will fix.